index.html

```diff
@@ -12,40 +12,289 @@
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li class="link-item"><a href="#" class="links"><i class="fa-brands fa-reddit-alien"></i></a></li>
+            </ul>
+        </section>
+        <!-- end of landing page -->
+
+    <!-- main -->
+        <section class="main">
+            <div id="recent-work" class="recent-work">
+                <h1 class="title">My recent works</h1>
+                <hr class="recent-work-line">
+            </div>
+            <div class="image-placeholder">
+                <img class="imge" src="assets/img/imageplaceholder.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <div class="teck-stacks">
+                    <ul class="ul-teck-stacks">
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">css </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">html </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Bootstrap </button></li>
+                        <li class = "teck-stacks-item"><button type="button" class="btn-tech-stacks">Ruby </button></li>
+                    </ul>
+                </div>
+                <div class="btn-see-project">
+                    <button type="button" class="">See Project </button>
+                </div>
+            </div>
+
+            <div class="recent-works">
+                <div class="card-container">
+                    <div class="work-card">
+                        <div class="work-card-body">
+                            <h1>Professional Art print Data</h1>
+                            <p class="wprk-description">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <div class="work-tech-stack">
+                                <ul class="ul-links">
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">html</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">bootstrap</button></li>
+                                    <li class="t-stack"><button type="button" class="btn-tech-stacks">Ruby</button></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="work-card-footer">
+                        <button type="button" class="">See Project </button>
```

Comment:
- [ ]  Kindly add hover effect to button so it looks like button in the figma design